.github/workflows/ci.yml

```
@@ -15,13 +15,23 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
-        django-version: ["3.2", "4.1", "4.2", "main"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
+        django-version: ["3.2", "4.1", "4.2", "5.0", "main"]
```

Break into one-per-line to reduce future diff noise:

```suggestion
        python-version:
        - "3.7"
        - "3.8"
        - "3.9"
        - "3.10"
        - "3.11"
        - "3.12"
        django-version:
        - "3.2"
        - "4.1"
        - "4.2"
        - "5.0"
        - "main"
```

---

[93d5ef2](https://github.com/jschneier/django-storages/pull/1331/commits/93d5ef2d5e936d5b82c05ca0815860cbdb2cefcb)